docs/software-architecture-and-design.md

```diff
@@ -141,6 +142,8 @@ The `TbotsProto::RobotStatus` protobuf message contains information about the st
 * The capacitor charge on the robot
 * The temperature of the dribbler motor
 
+The message also contains information on whether [Estop](#estop) should be activated. 
+
```

Comment:
I think this is out of date, as well as some other comments in this doc

Reply:
adjusted doc